table/block_based_table_reader.cc

```diff
@@ -2314,12 +2314,16 @@ InternalIterator* BlockBasedTable::NewIterator(
 }
 
 InternalIterator* BlockBasedTable::NewRangeTombstoneIterator(
-    const ReadOptions& /* read_options */) {
+    const ReadOptions& read_options) {
```

Comment:
I guess it's fine to have `NewRangeTombstoneIterator` functions return a `FragmentedRangeTombstoneIterator*`

Reply:
I was planning on doing this once I added flush/compaction support, but it probably doesn't hurt to do now.

Reply:
As discussed offline, I'll punt this cleanup to another PR, since there are some other associated cleanup items along with it, so those should probably not be lumped in here.